<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparatorWidget</class>
 <widget class="QWidget" name="ComparatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>711</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="margin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="rightLineNumbers">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(213, 213, 213);
color: rgb(150, 150, 150);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="rightText">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QScrollBar" name="leftHorizontalScrollBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QScrollBar" name="rightHorizontalScrollBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="rightFileName">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rightSelectFileButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="compareButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="leftLineNumbers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(213, 213, 213);
color: rgb(150, 150, 150);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="leftText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoNone</set>
        </property>
        <property name="tabChangesFocus">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="leftFileNameLayout">
      <item>
       <widget class="QLineEdit" name="leftFileName">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="leftSelectFileButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
